### DISCUSSION TITLE
Don't display +0 change in statistics

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#27421] [Ideas] [answered]
Hello, the stats change animation is a great addition! However maybe it should only happen when there is a change to be displayed. Right now the animation plays even for a +0 change after a play.

Thank you for considering my request!

**Comment by vatei** [comment]
Upping this because I just thought it might also be okay to simply play the animation faster for a +0 change (and it's still kinda bugging me)

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/29512

**Reply by vatei** [reply]
> #29512

If I understand correctly it's just removing the "+" right?
I might not have explained clearly but I was suggesting that the stat update was happening too often because it also plays the animation when no change is happening (so 0); so it might be better to only play it when the change is superior to 0
But on the other hand playing it all the time is feedback that something is happening so yeah not sure what the right approach is here
